rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_14.json

```diff
@@ -0,0 +1,1010 @@
+{
+    "rule-19-14-a": {
+        "Section": 19,
+        "Rule": 14,
+        "Test": "a",
+        "test_description": "System 11, no return or relief in the proposed, no return or relief in the baseline - PASS",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "19-14",
+            "ruleset_reference": "G3.1.2.8.1",
+            "rule_description": "For baseline system types 1-8 and 11-13, if return or relief fans are specified in the proposed design, the baseline building design shall also be modeled with fans serving the same functions and sized for the baseline system supply fan air quantity less the minimum outdoor air, or 90% of the supply fan air quantity, whichever is larger.",
+            "applicable_rmr": "Baseline RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "primary_rule": "Yes",
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Required Building Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "VAV Air Terminal 1",
+                                                        "is_supply_ducted": true,
+                                                        "type": "VARIABLE_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 11"
```

Comment:
```primary_airflow``` key is missing here. In the ```get_zone_supply_return_exhaust_relief_terminal_fan_power_dict``` function, if the key isn't available, ```primary_airflow``` assumes to be zero and the sum of ```primary_airflow``` values equals 0, there is an ```assert_``` error msg.

Reply:
Added primary_airflow